pkg/stanza/operator/input/file/finder_test.go

```diff
@@ -136,8 +136,8 @@ func TestFinder(t *testing.T) {
 			expected := absPath(tempDir, tc.expected)
 
 			for _, f := range files {
-				require.NoError(t, os.MkdirAll(filepath.Dir(f), 0755))
-				require.NoError(t, ioutil.WriteFile(f, []byte(filepath.Base(f)), 0755))
+				require.NoError(t, os.MkdirAll(filepath.Dir(f), 0755))                  // #nosec G601: permissions allowable for test
```

Comment:
Or 644

Reply:
I was able to reduce these to `0700`, which `gosec` is ok with. The test fails otherwise, though I don't understand why `ioutil.WriteFile` requires execution permission.
```
Error Trace:	finder_test.go:140
Error:      	Received unexpected error:
            	open /var/folders/4s/p4v5vql91m3dldy_q1t97zkh0000gn/T/411659339/a/1.log: permission denied
```